src/server/storage.ts

```diff
@@ -70,24 +48,13 @@ export async function fetchCodeManifest(info: MinimalJobInfo) {
 }
 
 async function storeFile(filePath: string, info: MinimalStudyInfo, file: File) {
-    if (USING_S3_STORAGE) {
-        await storeS3File(info, file.stream(), filePath)
-    } else {
-        const dirName = path.join(getUploadTmpDirectory(), path.dirname(filePath))
-        fs.mkdirSync(dirName, { recursive: true })
-        const buffer = await file.arrayBuffer()
-        await fs.promises.writeFile(path.join(dirName, path.basename(filePath)), Buffer.from(buffer))
-    }
-}
-
-export async function storeStudyCodeFile(info: MinimalJobInfo, file: File) {
-    await storeFile(pathForStudyJobCodeFile(info, file.name), info, file)
+    await storeS3File(info, file.stream(), filePath)
 }
 
 export async function storeStudyResultsFile(info: MinimalJobResultsInfo, file: File) {
     return await storeFile(pathForStudyJobResults(info), info, file)
 }
 
-export async function storeStudyDocumentFile(info: MinimalStudyInfo, fileType: StudyDocumentType, file: File) {
-    await storeFile(pathForStudyDocuments(info, fileType, file.name), info, file)
+export async function fetchStudyResultsFile(info: MinimalJobResultsInfo) {
+    return await fetchFile(pathForStudyJobResults(info))
 }
```

Comment:
this can be left for later, but the all storage is now s3, so these methods could be moved into s3.ts, OR s3.ts into here

Reply:
yea lets clean this stuff up later to reduce scope in this

Reply:
I added a TODO